packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js

```diff
@@ -26,10 +26,12 @@ import { Field, reduxForm, formValueSelector } from 'redux-form';
 import store from 'store';
 
 import * as markers from './SearchForm.markers';
+import { trackFormInput } from './SearchForm.track';
 import VirtSelect from '../common/VirtSelect';
 import * as jaegerApiActions from '../../actions/jaeger-api';
 import { formatDate, formatTime } from '../../utils/date';
 import reduxFormFieldAdapter from '../../utils/redux-form-field-adapter';
+import { DEFAULT_OPERATION, DEFAULT_LIMIT, DEFAULT_LOOKBACK } from '../../constants/search-form';
```

Comment:
`../../constants/etc...` is out of order

Reply:
We are. My remark refers to the order of the lines.

Reply:
no auto-formatter for order? Go and Java both have that for imports.

Reply:
Not in prettier, which is the auto-formatter we're using. There might be something around that fits the bill, thought...